documentation/src/main/asciidoc/topics/code_examples/HotRodClientInstance.java

```diff
@@ -0,0 +1,12 @@
+org.infinispan.client.hotrod.configuration.ConfigurationBuilder cb
+      = new org.infinispan.client.hotrod.configuration.ConfigurationBuilder();
+cb.tcpNoDelay(true)
+  .statistics()
+      .enable()
+      .jmxEnable()
+      .jmxDomain(org.infinispan)
+      .jmxName()
+  .addServer()
+      .host("127.0.0.1")
+      .port(11222);
+RemoteCacheManager rmc = new RemoteCacheManager(cb.build());
```

Comment:
Example client config where statistics are enabled, a marshaller specified and server added. Not entirely the same as the .properties file as I think providing an example with all those options configured would make the config far more complicated then it actually is.
```java
cb
  .marshaller(new org.infinispan.commons.marshall.ProtoStreamMarshaller())
  .statistics()
    .enable()
    .jmxDomain("org.infinispan")
  .addServer()
     .host("127.0.0.1")
     .port(11222);
```